android/app/src/main/java/org/fossasia/openevent/data/repository/RealmDataRepository.kt

```diff
@@ -510,6 +514,22 @@ class RealmDataRepository private constructor(val realmInstance: Realm) {
         }
     }
 
+    //Feedbacks section
+
+    private fun saveFeedbacksInRealm(feedback: List<Feedback>) {
+        val realm = Realm.getDefaultInstance()
+        realm.executeTransaction({ realm1 -> realm1.insertOrUpdate(feedback) })
+        realm.close()
```

Comment:
Why creating a new Realm?

Reply:
realm instance may already be closed which would make it unusable also that's how it was done for discount codes, tracks, faqs there